### DISCUSSION TITLE
Install web components on Quasar/Vue

### DISCUSSION THREAD

**Discussion by dgateles** [quasarframework/quasar#13481] [General - Components / Directives / etc]
Hello!

I'm trying to load a custom web component made with stenciljs on a Quasar/Vue/Vite project, but without success. How can i do that? On Vue 3/Vue CLI i have do successfully, but with Quasar/Vue3/Vite, at this moment, i have tryid with some methods but none has worked.

Thanks!

**Comment by MilosPaunovic** [comment]
Please elaborate a little more. What does it mean without success - there is an error or something else?
